src/db_reader.rs

```diff
@@ -74,18 +74,10 @@ impl ReadSnapshot for CheckpointState {
         Arc::clone(&EMPTY_TABLE)
     }
 
-    fn wal(&self) -> Arc<KVTable> {
-        Arc::clone(&EMPTY_TABLE)
-    }
```

Comment:
to allow dirty read, wal is not needed any more since we've put the uncommitted data inside memtable, and bounce up the `last_committed_id` to make this write to be visible. wal is considered as write-only after start up.